### PR TITLE
Disable/Enable for tabs

### PR DESCRIPTION
**Pull request by benhartouz**
i'm currently working on the project that i used native base as UI component.
Basically i want to disable and enable tabs when user interacting. and i think that will be a good feature for the project.

#2600 

```
import React, { Component } from 'react';
import { Container, Header, Content, Tab, Tabs } from 'native-base';
import Tab1 from './tabOne';
import Tab2 from './tabTwo';
import Tab3 from './tabThree';
export default class TabsExample extends Component {
  render() {
    return (
      <Container>
        <Header hasTabs />
        <Tabs>
          <Tab heading="Tab1" disabled={true}>
            <Tab1 />
          </Tab>
          <Tab heading="Tab2" disabled={true}>
            <Tab2 />
          </Tab>
          <Tab heading="Tab3" disabled={false}>
            <Tab3 />
          </Tab>
        </Tabs>
      </Container>
    );
  }
}
```

### CODE REVIEW

File: src/basic/Tabs/DefaultTabBar.js [hunk 1]
```diff
@@ -5,152 +5,169 @@ import _ from "lodash";
...
-          style={[tabUnderlineStyle, { left }, this.props.underlineStyle]}
-        />
-      </TabContainer>
-    );
-  }
+        const left = this.props.scrollValue.interpolate({
+            inputRange: [0, 1],
+            outputRange: [0, containerWidth / numberOfTabs]
+        });
+        console.log("tabs:", this.props.tabs);
```
Comment (sankhadeeproy007) [inline src/basic/Tabs/DefaultTabBar.js]: Remove the log please.

File: src/basic/Tabs/DefaultTabBar.js [hunk 2]
```diff
@@ -5,152 +5,169 @@ import _ from "lodash";
...
+        } else {
+            return (
+                <Button
+                    style={{ flex: 1 }}
+                    key={_.random(1.2, 5.2)}
+                    onPress={() => {
+                        if (disabled === true) {
+                            return;
+                        }
+                        onPressHandler(page);
```
Comment (sankhadeeproy007) [inline src/basic/Tabs/DefaultTabBar.js]: Would something like this work?
```
onPress={() => disabled ? {} : onPressHandler(page)}
```

Just to make the code a bit more concise.

Comment (benhartouz) [reply src/basic/Tabs/DefaultTabBar.js]: I improved the code and I resolved the conflict thanks for your review.

File: src/basic/Tabs/DefaultTabBar.js (hunk 1 above)
Comment (benhartouz) [reply src/basic/Tabs/DefaultTabBar.js]: yeah i'm sorry

**Comment by benhartouz** [comment]
@sankhadeeproy007 I think now you can merger the pull request.

**Comment by sankhadeeproy007** [comment]
@benhartouz Any chance you can squash the commits into 1?

**Comment by RohanTalip** [comment]
@sankhadeeproy007 you should be able to squash the commits on your end: https://help.github.com/en/articles/about-pull-request-merges

See also the "Squash and merge" option on https://help.github.com/en/articles/merging-a-pull-request .

**Comment by shivrajkumar** [comment]
The latest version has a `disabled ` prop for this.

**Pull request closed without merging**